### PR TITLE
AGENT-169: AgentClusterInstall asset

### PR DESCRIPTION
**Pull request by rwsu**
Creates the asset for AgentClusterInstall.

There is a partial implementation of the translation of
install-config.yaml to AgentClusterInstall.

### CODE REVIEW

File: pkg/asset/agent/manifests/agentclusterinstall.go [hunk 1]
```diff
@@ -1 +1,111 @@
...
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+)
+
+var (
+	agentClusterInstallFilename = filepath.Join(clusterManifestDir, "agent-cluster-install.yml")
```
Comment (bfournie) [inline pkg/asset/agent/manifests/agentclusterinstall.go:18]: Nit - using "yml" for the extension is consistent with infraenv.yml but different than pull-secret.yaml. Either is fine, just wondering if we should be consistent.

Comment (bfournie) [inline pkg/asset/agent/manifests/agentclusterinstall.go:18]: Nit - using "yml" for the extension is consistent with infraenv.yml but different than pull-secret.yaml. Either is fine, just wondering if we should be consistent.

Comment (rwsu) [reply pkg/asset/agent/manifests/agentclusterinstall.go:18]: I think we've been using .yaml. Thank and good catch!

File: pkg/asset/agent/manifests/agentclusterinstall_test.go [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+	"github.com/openshift/installer/pkg/asset/mock"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// func TestAgentClusterInstall_Generate(t *testing.T) {
+
+// 	installConfig := &installconfig.InstallConfig{
```
Comment (andfasano) [inline pkg/asset/agent/manifests/agentclusterinstall_test.go:20]: nit: also t.Skip can be used

**Pull request merged**